packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -177,6 +177,8 @@ export class ApolloServerBase<
 
     this.parseOptions = parseOptions;
     this.context = context;
+    this.experimental_approximateDocumentStoreMiB =
```

Comment:
Ah, good catch! I would have thought that TS would give a noUnusedLocals error on the declaration of experimental_approximateDocumentStoreMiB but its existence affects the contents of `requestOptions`.